### DISCUSSION TITLE
Why video upload to https://video.bsky.app not to PDS?

### DISCUSSION THREAD

**Discussion by lovemyliwu** [bluesky-social/atproto#3060] [How to do it?]
I want keep video data on my own PDS, is that possible?

Currently, how can I config the video service related endpoint through PDS?

**Comment by yamarten** [comment]
In any case, the video is stored in your PDS. If you use the `video.bsky.app` API, it will only delay the upload of the video to the PDS until it is ready on CDN.

You can also upload a video directly to your PDS, just like an image.

https://bsky.app/profile/samuel.bsky.team/post/3l6supolf6w25

**Reply by lovemyliwu** [reply]
that's not true, the blob is stored in bluesky video service, PDS only saved a post with the `cid`.

**Reply by yamarten** [reply]
Can you show me an example of a post where the video is not saved in PDS?

I tried a few posts, but I couldn't find anything that I couldn't getBlob.
